pkgs/applications/audio/airwindows/default.nix

```
@@ -0,0 +1,62 @@
+{ stdenv, fetchFromGitHub, requireFile, unzip, cmake }:
+
+stdenv.mkDerivation rec {
+  name = "airwindows-${version}";
+  version = "2020-05-25";
+
+  src = fetchFromGitHub {
+    owner = "airwindows";
+    repo = "airwindows";
+    rev = "897c0830e603a950e838a3753f89334bdf56bb71";
+    sha256 = "1mh7c176fdl5y5ixkgzj2j2b7qifp3lgnxmbn6b7bvg3sv5wsxg3";
+  };
+
+  vst-sdk = stdenv.mkDerivation rec {
+    name = "vstsdk366_27_06_2016_build_61";
+    src = requireFile {
+      name = "${name}.zip";
+      url = "https://www.steinberg.net/sdk_downloads/vstsdk366_27_06_2016_build_61.zip";
+      sha256 = "05gsr13bpi2hhp34rvhllsvmn44rqvmjdpg9fsgfzgylfkz0kiki";
+    };
+    nativeBuildInputs = [ unzip ];
+    installPhase = "cp -r . $out";
+  };
+
+  airwindows-ports = stdenv.mkDerivation rec {
+    name = "airwindows-ports";
+    src = fetchFromGitHub {
+      owner = "ech2";
+      repo = "airwindows-ports";
+      rev = "0.4.0";
+      sha256 = "1ya4qbc63sb52nzisdapsydrnnpqnjsl5kgxibbr3dxf32474g89";
+    };
+    installPhase = "cp -r . $out";
+  };
```

You can add the unfree license here and keep the main derivation MIT.
```suggestion
    meta.license = stdenv.lib.licenses.unfree;
  };
```

---

Thanks for the feedback, I'm working on it.
I think you mean line 23, in vst-sdk, right?

---

Err yea, apologies.